lib/Backend/SccLiveness.cpp

```diff
@@ -165,8 +165,6 @@ SCCLiveness::Build()
                 if (labelInstr->isOpHelper == (this->lastOpHelperLabel != nullptr)
                     && lastLabelInstr && labelInstr->isOpHelper == lastLabelInstr->isOpHelper)
                 {
-                    // No such transition. Remove the label.
-                    Assert(!labelInstr->GetRegion() || labelInstr->GetRegion() == this->curRegion);
                     labelInstr->Remove();
```

Comment:
Removing this assert. It is possible that a block containing a label gets assigned a region and then dead coded due to unreachable block elimination. With try finally having early exit, blocks can become unreachable later on after first pass of region assignment is done. See test6() in test/EH/tryfinallytests.js